build.gradle.kts

```
@@ -151,3 +151,24 @@ tasks.register<org.openapitools.generator.gradle.plugin.tasks.GenerateTask>("ope
   ))
   library.set("resttemplate")
 }
+
+tasks.register<org.openapitools.generator.gradle.plugin.tasks.GenerateTask>("openApiGenerateOrganization") {
+  group = "openapi"
+  description = "description"
+
+  generatorName.set("java")
+  remoteInputSpec.set("https://raw.githubusercontent.com/pagopa/p4pa-organization/refs/heads/$targetEnv/openapi/generated.openapi.json")
+  outputDir.set("$projectDir/build/generated")
+  apiPackage.set("it.gov.pagopa.pu.p4pa-organization.controller.generated")
+  modelPackage.set("it.gov.pagopa.pu.p4pa-organization.model.generated")
```

let's change this package also in the other openApi generation
```suggestion
  modelPackage.set("it.gov.pagopa.pu.p4pa-organization.dto.generated")
```